### DISCUSSION TITLE
Expand supported data (like assignees, labels, etc) for draft issues

### DISCUSSION THREAD

**Discussion by azenMatt** [community/community#4255] [Projects and Issues]
Being able to quickly add drafts to a Project is great, but similar to Notes on the old project board I would like to be able to assign someone on my team to them and add other details, like a description.

**Comment by BirgitPohl** [comment]
Very much needed. 
The drafts are not assigned to a specific repository.
I don't want my PO to know which feature goes in which repository. This is the development team's responsibility when they split items into separate items. Also, my teams are working on very technical topics.

In addition to the request, I very much vote for uploading images. Currently, it is possible to add images via markdown, but our POs are not technical. They want a simple WYSIWG window, where they can upload an image inside a draft. The same features, we already have for issues that are bound to repositories and this includes uploading images.

Edit: Now I onboarded our designer into using beta boards. We need visuals (images in drafts) :) Because she is now taking a get-around on creating a placeholder repo, so she convert the draft to issues to get all the features, she needs.

**Comment by krantzinator** [comment]
+1. While we can assign users to Drafts now, there are still several improvements that we'd like to see, to make them less confusing in daily use.
For one, we need to be able to call it not-a-Draft 😄 We use `Drafts` for issues that are not assignable to a single/any repo, and having to have it called `Draft` and have the `draft` circle icon next to the issue name is a paper-cut.
Additionally, would love to see the improvements mentioned in https://github.com/orgs/community/discussions/16610 and https://github.com/community/community/discussions/7378.
